lib/cdo/hamburger.rb

```diff
@@ -115,30 +111,19 @@ def self.get_hamburger_contents(options)
       entry[:title] = I18n.t("#{loc_prefix}#{entry[:title]}")
     end
 
-    if options[:language] == "en"
-      if options[:user_type] == "teacher"
-        teacher_entries << {
-          title: I18n.t("#{loc_prefix}professional_learning"),
-          url: CDO.studio_url("/my-professional-learning"),
-        }
-      end
+    if options[:user_type] == "teacher"
+      teacher_entries << {
+        title: I18n.t("#{loc_prefix}professional_learning"),
+        url: CDO.studio_url("/my-professional-learning"),
+      }
+    end
```

Comment:
Since this will only add these entries if the user is a teacher, I think these can just be added to `teacher_entries` on line 91